### ISSUE TITLE
Veteran ID – copy function not accessible to assistive technology (AT)

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12097]
The role of the Veteran ID cannot be determined with assistive technology programs. Visually there is an icon indicating this is a copy function as well as a mouse over that reads Click to Copy. (New: 08/22/2019)
